Sources/TuistGenerator/Generator/SchemesGenerator.swift

```diff
@@ -572,7 +572,7 @@ final class SchemesGenerator: SchemesGenerating {
     private func commandlineArgruments(_ arguments: [String: Bool]) -> [XCScheme.CommandLineArguments.CommandLineArgument] {
         arguments.map { key, enabled in
             XCScheme.CommandLineArguments.CommandLineArgument(name: key, enabled: enabled)
-        }
+        }.sorted { $0.name < $1.name }
```

Comment:
Does order matter when generating schemes?

Reply:
Wondering, why there are differences in order when generating subsequent times, can we identify the root cause of the problem?

Reply:
Generally it does, but this change is not regressing as the order is currently indeterministic. Please check https://github.com/tuist/tuist/issues/791 for more details.

Reply:
@lakpa Arguments are stored as a dictionary in the manifest, that's why they're shuffled.

```
public struct Arguments: Equatable, Codable {
    public let environment: [String: String]
    public let launch: [String: Bool]

    public init(environment: [String: String] = [:],
                launch: [String: Bool] = [:]) {
        self.environment = environment
        self.launch = launch
    }
}
```